plugins/plugin-site/src/components/Plugin.jsx

```diff
@@ -27,7 +27,7 @@ Developers.propTypes = PluginDevelopers.propTypes;
 function Plugin({plugin: {name, title, stats, labels, excerpt, developers, buildDate, releaseTimestamp}}) {
     let progress = 0;
     let color =
```

Comment:
computation of color should go to the PluginHealthScore component

Reply:
Learning some implementation of healthscore in algolia, then will make some changes in code